common.mk

```diff
@@ -566,7 +568,7 @@ OPTEE_OS_COMMON_FLAGS ?= \
 	CFG_TEE_CORE_LOG_LEVEL=$(CFG_TEE_CORE_LOG_LEVEL) \
 	DEBUG=$(DEBUG) \
 	CFG_TEE_BENCHMARK=$(CFG_TEE_BENCHMARK) \
-	CFG_IN_TREE_EARLY_TAS=trusted_keys/f04a0fe7-1f5d-4b9b-abf7-619b85b4ce8c
+	CFG_IN_TREE_EARLY_TAS=$(CFG_IN_TREE_EARLY_TAS)
```

Comment:
I have to escape the space char separator in my shell command line to get to build to work.
Without seperartor:
```shell
$ make CFG_IN_TREE_EARLY_TAS="pkcs11/fd02c9da-306c-48c7-a49c-bbd827ae86ee trusted_keys/f04a0fe7-1f5d-4b9b-abf7-619b85b4ce8c"
make[1]: *** No rule to make target 'trusted_keys/f04a0fe7-1f5d-4b9b-abf7-619b85b4ce8c'.
```
With the escape:
```shell
$ make CFG_IN_TREE_EARLY_TAS="pkcs11/fd02c9da-306c-48c7-a49c-bbd827ae86ee\ trusted_keys/f04a0fe7-1f5d-4b9b-abf7-619b85b4ce8c"
... (runs ok)
```

Using quotes (simple or double) at line 571 seems to address the issue:
```patch
-	CFG_IN_TREE_EARLY_TAS=$(CFG_IN_TREE_EARLY_TAS)
+	CFG_IN_TREE_EARLY_TAS="$(CFG_IN_TREE_EARLY_TAS)"
```

Reply:
Good catch! It shows that I did not test the very example that I gave :open_mouth:

Updated.